.circleci/config.yml

```diff
@@ -18,7 +18,8 @@ jobs:
       jdk_tag: "21.0"
       localstack_tag: "3.0"
       postgres_tag: "16.1"
-      java_options: "-Xmx4096m -XX:ParallelGCThreads=2 -XX:ConcGCThreads=2 -Djava.util.concurrent.ForkJoinPool.common.parallelism=2 -Dorg.gradle.daemon=true -Dkotlin.compiler.execution.strategy=in-process -Dorg.gradle.workers.max=4"
+      java_options: "-Xmx4096m -XX:ParallelGCThreads=2 -XX:ConcGCThreads=2 -Djava.util.concurrent.ForkJoinPool.common.parallelism=2 -Dorg.gradle.daemon=true -Dkotlin.compiler.execution.strategy=in-process -Dorg.gradle.workers.max=1"
+      resource_class: medium+
```

Comment:
Increased this due to memory constraints running on medium (4Gb mem) to handle heap size https://circleci.com/docs/configuration-reference/#resourceclass

Reply:
can we change the circle instance type?
👍 anyway